conf/modules.config

```
@@ -465,6 +465,34 @@ process {
         ]
     }
 
+    withName: ANGSD_DOCOUNTS {
+        ext.args = [
+            "-iCounts 1",
+            "-r ${params.contamination_estimation_angsd_chrom_name}:${params.contamination_estimation_angsd_range_pos1}-${params.contamination_estimation_angsd_range_pos2}",
```

```suggestion
            "-r ${params.contamination_estimation_angsd_chrom_name}:${params.contamination_estimation_angsd_range_from}-${params.contamination_estimation_angsd_range_to}",
```
position_from/position_to would be clearer name, since we take a range between the positions.